File: Baekjoon/Class1/31.2577 숫자의 개수.py
import sys
input = sys.stdin.readline

a = int(input())
b = int(input())
c = int(input())
product = a * b * c
count = [0] * 10
while product != 0:
    remainder = product % 10
    count[remainder] += 1
    product //= 10
print("\n".join(map(str, count)))

'''
a = int(input())
b = int(input())
c = int(input())
product = str(a * b * c)
for i in range(10):
    print(product.count(str(i)))
'''

File: Baekjoon/Class1/31.2577 숫자의 개수.cpp
#include <iostream>
using namespace std;

int main(void) {
	ios_base::sync_with_stdio(false);
	cin.tie(NULL);

	int a, b, c, product, remainder;
	int count[10] = { 0 };
	cin >> a >> b >> c;
	product = a * b * c;
	while (product != 0) {
		remainder = product % 10;
		count[remainder]++;
		product /= 10;
	}
	for (int i = 0; i < 10; i++) {
		cout << count[i] << '\n';
	}
	return 0;
}